test/Async.test.ts

```diff
@@ -187,12 +187,12 @@ describe("Async", () => {
     })
 
     describe("never", () => {
-        it("never resolves", async () => {
+        it("never resolves", () => {
             // arrange
             vi.useFakeTimers()
             const f = vi.fn()
             // act
-            pipe(Async.never, Async.tee(f), Async.start)
+            void pipe(Async.never, Async.tee(f), Async.start)
```

Comment:
ahhh.. right. this makes way more sense than what I was doing 😬

Reply:
The lint rules figured it out, not me 😄